project1/css/index4FlexboxChallenge.css

```diff
@@ -0,0 +1,82 @@
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+header {
+    display: flex;
+    width: 100%;
+}
+
+header nav {
+    flex-grow: 1;
+}
+  
+header nav ul {
+    display: flex;
+    list-style: none;
+    justify-content: space-around;
+}
+
+.container a {
+    text-decoration: none;
+    color: gray;
+}
+
+.cta {
+    display: flex;
+    justify-content: space-around;
+}
+
+.cta .cta-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-size: 72px;
+    text-align: center;
+    justify-content: space-around;
+}
+
+.cta img {
+    display: flex;
+    align-self: center;
+}
+
+.cta .cta-text button{
+    font-size: 20px;
+    color: gray;
+    background-color: white;
+    border: 1px solid gray;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 10px;
+}
+
+.main-content {
+    border-top: 1px solid blue;
+    border-bottom-width: 80%;
+    border-bottom: 1px solid blue;
+    border-bottom-width: 80%;
+    margin: 20px 0;
```

Comment:
Try setting this to flex display with a column direction and align its items at center to remove the margin and style with pure flexbox. :)